src/main/java/org/assertj/core/api/recursive/comparison/RecursiveComparisonConfiguration.java

```diff
@@ -141,6 +153,15 @@ public void ignoreFieldsMatchingRegexes(String... regexes) {
     return ignoredFields;
   }
 
+  /**
+   * Returns the set of the object under test field types to ignore in the recursive comparison.
+   *
+   * @return the set of the object under test field types to ignore in the recursive comparison.
+   */
+  public Set<Class<?>> getIgnoredFieldsForTypes() {
```

Comment:
rename to `getIgnoredTypes` as this is not returning fields (or `FieldLocation`)